libs/device/models/src/lib/device-communication-event.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const ApiSerialPortEvents = {
+  DataReceived: "api-serial-port-data-received",
+} as const
+
+export type ApiSerialPortEvent =
+  (typeof ApiSerialPortEvents)[keyof typeof ApiSerialPortEvents]
+
+export const ApiSerialPortToRendererEvents = {
+  Connected: "api-serial-port-connected",
+  Disconnected: "api-serial-port-disconnected",
+  InitializationFailed: "api-serial-port-initialization-failed",
```

Comment:
🟡 
 suggest changing the state name
```suggestion
  ConnectionFailed: "api-serial-port-connection-failed",
```

This change more accurately reflects the nature of the error, which occurs during an attempt to establish a connection via the serial port interface rather than specifically during the initialization process. 

This modification should more appropriately describe the situation where an error occurs during an attempt to establish a connection with the serial port interface, not necessarily during the initialization process.